test/verify/check-metrics

```diff
@@ -811,6 +818,15 @@ class TestCurrentMetrics(MachineCase):
             b.wait_in_text(".pf-c-tooltip", "GiB total")
             b.mouse("#current-swap-usage", "mouseleave")
 
+        # Test link to user services
+        m.execute("su - admin -c \"export XDG_RUNTIME_DIR=/run/user/$(id -u admin); systemd-run --user --collect --slice cockpittest --unit mem-hog3 awk "
+                  """\'BEGIN { x = sprintf(\\"%""" + str(100) + """000000s\\",\\"\\"); system(\\"touch /tmp/hogged; sleep infinity\\") }\'\"""")
+        m.execute("while [ ! -e /tmp/hogged ]; do sleep 1; done")
+        b.wait_text("table[aria-label='Top 5 memory services'] tbody tr:nth-of-type(2) td[data-label='Service']", "mem-hog3")
```

Comment:
Same here, we should really expect it at the top of the list, and make sure we clean up previous memory allocations from the session. Most test failures happened because mem-hog3 is at the top of the list.